metaversefile-api.js

```diff
@@ -841,11 +856,18 @@ export default () => {
   getAvatarHeight(obj) {
     return getHeight(obj);
   },
+  getGfxSettingJSON(setting) {
+    if (setting) return JSON.parse(localStorage.getItem('GfxSettings'))?.[setting];
+    return JSON.parse(localStorage.getItem('GfxSettings'));
+  },
+  setGfxSetting(quality) {

```

Comment:
1) Is there ever a need for an app to be setting this? It's an abusable feature (imagine a world that forces quality against your settings) so I don't want to make it easy for apps to override this.

2) Any such getter usually needs an event system (`EventTarget`). Otherwise how are you going to update when it's changed?

3) This should not be owned by `metaversefile`. These settings should be in `settings.js`, which exports an `EventTarget` and abstracts the local storage aspect, rather than littering it across many files in different directories. And the UI should use `settings.js` so it's one point of control, not different apps with different ideas of what setting are.